draft-regext-simple-contact.md

```diff
@@ -88,135 +91,61 @@ The "kind" JSON value is a string, that is either "individual", "role" or
 
     "kind" : "role"
 
+There is no equivalent to the "role" value in either jCard or JSContact,
+though role entities are common in RDAP registries.
 
 ## Names
 
-Names may be expressed as unstructured text and as structured data.
-When names are given as structured data, it is RECOMMENDED to also
-give unstructured names. This may require servers which store names
-in structured data to synthesize the unstructured names. 
-Servers which store only unstructured names SHOULD NOT attempt to synthesize 
-and provide structured names from those values, because of the difficulty 
-of doing this correctly for all types of names.
-
 Names can be expressed for each kind of the entity, as described in the
 "kind" string. When describing an "individual", the name of the individual's
 role and organization may also be expressed. When describing a "role", the
 name of the role's organization may also be expressed. It is NOT RECOMMENDED
 to express the name of a role or individual when the kind is "organization".
 
-Each type of name is expressed as a JSON array in which each item is an
-object with the following optional members:
-
-* "name" - the unstructured name as a string
-* "lang" - see above
-* "masked" - see above
-* "parts" - a JSON object that varies for each type of name.
-
-A> The co-authors of this specification are skeptical of the need for
-A> structured names in RDAP. None of the references in Section 1 indicate
-A> that names are collected or published structurally. Additionally,
-A> attempts to structure names for all contexts and languages is unlikely
-A> to yield a complete solution 
-A> (see https://mailarchive.ietf.org/arch/msg/calsify/3MR_gLO-1NuyQvE8DIl88KNglSs/).
```

Comment:
Like this thoughtful reduction given the Sisyphean task it otherwise would be. However, wonder if this would create a gap for the scenarios in need to such structured data, and any advise we can give there? Do like that we have the "about" link concept to more richer/complex formats.

Reply:
The feedback from IETF 117 was pretty clear that structured names were not desired.